corehq/motech/repeaters/models.py

```diff
@@ -298,6 +299,28 @@ class Meta:
     def repeater(self):
         return Repeater.get(self.repeater_id)
 
+    def to_json(self):
+        repeater_dict = self.__dict__.copy()
+        options = repeater_dict.pop('options', None)
+
+        # Populating OptionValue attrs
+        for attr_tuple in inspect.getmembers(self.__class__):
+            if isinstance(attr_tuple[1], OptionValue):
+                attr = attr_tuple[0]
+                repeater_dict[attr] = options.get(attr) if attr in options else getattr(self, attr, None)
```

Comment:
`getattr(self, attr, None)` changes the state of `self.options`. Might be better to refactor the default value calculation logic int a method (`OptionValue.get_default`) and use `attr_tuple[1].get_default()`.

Also, if the default is `NOT_SET` then `getattr(self, attr, None)` may raise `AttributeError`. That does not appear to be handled.